src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/EvidenceUploadHandler.java

```diff
@@ -38,4 +51,85 @@ public List<CaseEvent> handledEvents() {
         return EVENTS;
     }
 
+    private CallbackResponse validateValues(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+        List<String> errors = new ArrayList<>();
+
+        if (caseData.getDocumentUploadWitness1() != null) {
+            List<Element<UploadEvidenceWitness>> dateList = caseData.getDocumentUploadWitness1();
+            dateList.forEach(date -> {
+                if (date.getValue().getWitnessOption1UploadDate().isAfter(LocalDate.now())) {
+                    errors.add("Invalid date: date entered must not be in the future.");
+                }
+            });
+        }
```

Comment:
Extract a common method for this test, since it's identical to all other ones.

This will make the method considerably shorter.

Also, replace LocalDate.now() with the Time bean (defined in TimeConfiguration, check other classes to see how to use it) as it will help with unit-testing.